docs/blog/0.6.md

```diff
@@ -0,0 +1,40 @@
+---
+date: 2022-07-19
+image: ../static/gif/Rill_0.6.0.gif
+---
+
+# Rill 0.6 – Introducing radically simple dashboards
+
+:::note
+⚡ Rill Developer is a tool that makes it effortless to transform your datasets with SQL and create powerful, opinionated dashboards. These are our release notes for the `0.6` release, still in Tech Preview.
+
+To [try out Rill Developer, check out these instructions](https://docs.rilldata.com/#pick-an-install-option) and [let us know over on Discord](https://bit.ly/3bbcSl9) if you encounter any problems or have ideas about how to improve Rill Developer!
+:::
+
+![Introducing radically simple dashboards](http://pkg.rilldata.com/assets/Rill_0.6.0.gif)
+
+## Radically simple dashboards
+
+Ever since our first `0.1` release, we've had countless people in the community ask us if we were planning to tie in more data visualization and exploration tools on top of our modeling functionality. `0.6` marks our first foray, starting with a simple drill-down dashboard bundled directly into Rill Developer. The dashboard tool enables you to define measures and dimensions on top of models, then provides a fast, powerful, opinionated way to drill down into your dimensions to uncover new trends. We think it still has some of the same reactive magic as our SQL GUI; we like to think of it as a conversation-fast un-dashboard.
```

Comment:
```suggestion
Ever since our first `0.1` release, we've had countless people in the community ask us if we were planning to add more data visualization and exploration tools on top of our modeling functionality. `0.6` marks our first foray, starting with a simple drill-down dashboard bundled directly into Rill Developer. The dashboard tool enables you to define measures and dimensions on top of models, then provides a fast, powerful, opinionated way to drill down into your dimensions to uncover new trends. We think it still has some of the same reactive magic as our SQL GUI; we like to think of it as a conversation-fast un-dashboard.
```